src/sanitizer.js

```diff
@@ -106,6 +107,10 @@ const WHITELISTED_ATTRS = [
   'href',
   'on',
   'placeholder',
+  /* Attributes added for amp-bind */
+  // TODO(kmh287): Add more whitelisted attributes for bind?
+  'text',
+  'class',
```

Comment:
Probably harmless, though only whitelisting the amp-bind attributes `[text]` and `[class]` would avoid the need for validating the value of `class`.